### ISSUE TITLE
Command "choose ok on next confimation" fails in 3.3.1

### ISSUE THREAD

**Issue by anzai** [SeleniumHQ/selenium-ide#392]
Command "choose ok on next confimation" fails in 3.3.1.
In previous versions, this command succeeded even if no confirmation appeared after the command.

When I start my test suite with this command, "Log" tab saids
```
chooseCancelOnNextConfirmation on ... Failed:
e is undefined"
```

Command "webdriver choose ok on visible confirmation"  has the same problem.

**Comment by mondiaz** [comment]
@tourdedave Do you know when this might get released? Guessing this will be part of the next version of Selenium IDE...

**Comment by tourdedave** [comment]
Yes, this will be available in the next release. Not sure when we’ll be
releasing, but soon.

בתאריך יום ה׳, 27 בספט׳ 2018 ב-18:52 מאת Monica Diaz <
notifications@github.com>:
